adapters/kagome/src/host_api/helpers.cpp

```diff
@@ -215,7 +219,11 @@ namespace helpers {
         // Initialize module factory
         auto instance_env_factory =
             std::make_shared<binaryen::InstanceEnvironmentFactory>(
-                trie_db, host_api_factory, header_repo, changes_tracker);
+                trie_db,
+                serializer,
+                host_api_factory,
+                header_repo,
+                changes_tracker);
```

Comment:
Is this build against a newer kagome version? Please update submodule (and hunter gate if necessary) accordingly.

Reply:
Done.